packages/web/src/components/layouts/HomeNew/HomeNew.tsx

```diff
@@ -0,0 +1,160 @@
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+        ],
+        numberOfUses: 5265,
+      },
+    },
+    {
+      title: 'unist-util-generated',
+      description: 'unist utility to check if a node is generated',
+      featuredSites: {
+        iconList: [
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+        ],
+        numberOfUses: 5265,
+      },
+    },
+    {
+      title: 'react-smooth',
+      description: 'is a animation library work on React',
+      featuredSites: {
+        iconList: [
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+        ],
+        numberOfUses: 5265,
+      },
+    },
+    {
+      title: 'unist-util-position',
+      description: 'unist utility to get the positional info of nodes',
+      featuredSites: {
+        iconList: [
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+        ],
+        numberOfUses: 5265,
+      },
+    },
+    {
+      title: 'vfile-message',
+      description: 'Create vfile messages',
+      featuredSites: {
+        iconList: [
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+          'https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg',
+        ],
+        numberOfUses: 5265,
+      },
+    },
+    {
+      title: 'Go to all Popular packages',
+      variant: 'toAll',
+    },
+  ];
+
+  const vulnerablePackages = [
+    {
+      title: 'disneyland.omsk.ru/signup',
+      vulnerablePackage: {
+        name: 'mdast-util-from-markdown',
+      },
+      variant: 'vulnerable',
+    },
+    {
+      title: 'disneyland.omsk.ru/signup',
+      vulnerablePackage: {
+        name: 'mdast-util-from-markdown',
+        moreCount: 1,
+      },
+      variant: 'vulnerable',
+    },
+    {
+      title: 'disneyland.omsk.ru/signup',
+      vulnerablePackage: {
+        name: 'mdast-util-from-markdown',
+      },
+      variant: 'vulnerable',
+    },
+  ];
+
+  return (
+    <>
+      <Hero suggestions={suggestions} />
+
+      <Container>
+        <CardGroups>
+          <CardGroup title='Popular search queries'>
+            <CardList cards={popularCards} />
```

Comment:
nit: We'll probably need separate components per variant (i.e. `<PopularWebsiteCard>` `<PopularPackageCard>`). This seems to align with how styles are currently structured for the cards. just taking note to fix this after drafting phase.